src/plugins/intel_gpu/tests/passes/select_preferred_formats_test.cpp

```diff
@@ -41,6 +41,7 @@ TEST(test_select_preferred_formats, setting_target_conv_format) {
     layout_optimizer lo(true);
     auto prog = program::build_program(engine, topology, build, false, true);
 
+    prog->get_node("conv1").get_output_layouts(false);
```

Comment:
Please leave a comment on the purpose of this line. For example...
`// Call calc_output_layout to set output layout. It is necessary because this test is running select_preferred_formats pass only.`

Reply:
Done.